src/HttpBuilder.elm

```diff
@@ -0,0 +1,475 @@
+module HttpBuilder
```

Comment:
Vendoring of https://github.com/lukewestby/elm-http-builder/pull/25 to be removed once it is ported.